docs/docs/cmd/spo/site/site-recyclebinitem-list.md

```diff
@@ -0,0 +1,40 @@
+# spo site recyclebinitem list
+
+Lists items from recycle bin
+
+## Usage
+
+```sh
+m365 spo site recyclebinitem list [options]
+```
+
+## Options
+
+`-u, --siteUrl <siteUrl>`
+: URL of the site for which to retrieve the recycle bin items
+
+`--type [type]`
+: type of items which should be retrieved (1 - list items, 3 - folder, 5 - files)
```

Comment:
I thought I shared it already but apparently I totally forgot it: could we please change the value to human-readable strings like `listItems`, `folders` and `files` instead of the cryptic numbers so that users don't have to lookup commands help each time they want to run the command? I'm so sorry I didn't catch it the first time or even earlier in the spec.

Reply:
no worries. Sure will do 👍

Reply:
ok changed to string which is compared/populated based on list of possible types. and also is used in autocomplete. I was fallowing the same approach as in CliIssueCommand. At least I hope so 🙂